common/constants.go

```diff
@@ -57,7 +57,7 @@ const (
 	AstraPrivateCloudName = "private"
 
 	ConnectorNeptuneCapability = "neptuneV1"
-	ConnectorRelayCapability   = "relayV1"
+	ConnectorV2Capability      = "connectorV2" // V2 refers specifically to Arch 3.0 connector and beyond
```

Comment:
I was going to keep the relayV1 capability around previously, but now that we pin the operator version in polaris, I figured it was fine to do it all in one fell swoop instead.

Reply:
I lied, going to merge this first, as the Polaris PR will need the new build version for its PR. It might screw over some people for 1-2 hours (those living on the cutting edge as I mentioned), but it's Friday and they should really just be using the pinned version that's in the Polaris repo anyway.